packages/wct-mocha/README.md

```diff
@@ -0,0 +1,159 @@
+[![Gitter](http://img.shields.io/badge/slack-join%20chat%20%E2%86%92-brightgreen.svg?style=flat-square)](https://polymer-slack.herokuapp.com/)
+
+`wct-mocha` makes testing your web components a breeze!
+
+You get a streamlined browser-based testing environment, designed to work with [web-component-tester](https://github.com/Polymer/tools/tree/master/packages/web-component-tester) or on its own.
+
+
+# Getting Started
+
+## Install mocha and wct-mocha as devDependencies
+
+```bash
+$ npm install --save-dev mocha
+$ npm install --save-dev wct-mocha
+```
+
+## Install `web-component-tester` or `polymer-cli` globally
+
+```bash
+$ npm install --global web-component-tester
+```
+
+## Run `wct`
+
+```bash
+$ wct --wct-package-name wct-mocha
+```
+
+## `.html` Suites
+
+Your test suites can be `.html` documents. For example,
+`test/awesomest-tests.html`:
+
+```html
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <script src="../node_modules/mocha/mocha.js"></script>
+  <script src="../node_modules/chai/chai.js"></script>
+  <script src="../node_modules/wct-mocha/browser.js"></script>
+</head>
+<body>
+  <awesome-element id="fixture"></awesome-element>
+  <script type="module">
+    import {AwesomeElement} from '../src/awesome-element.js';
+    suite('<awesome-element>', function() {
+      test('is awesomest', function() {
+        const element = document.getElementById('fixture');
+        chai.assert.instanceOf(AwesomeElement);
+        chai.assert.isTrue(element.awesomest);
+      });
+    });
+  </script>
+</body>
+</html>
+```
+
+## `.js` Suites
+
+Alternatively, you can write tests in separate `.js` sources. For example,
+`test/awesome-tests.js`:
+
+```js
+suite('AwesomeLib', function() {
+  test('is awesome', function() {
+    assert.isTrue(AwesomeLib.awesome);
+  });
+});
+```
+
+## Special Features
+
+### Web Server
+
+If you prefer not to use WCT's command line tool, you can also run WCT tests
+directly in a browser via a web server of your choosing.
+
+Make sure that WCT's `browser.js` is accessible by your web server, and have
+your tests load `browser.js`.
+
+#### Nested Suites
+
+To help support this case, you can also directly define an index that will load
+any desired tests:
+
+```html
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <script src="../node_modules/mocha/mocha.js"></script>
+    <script src="../node_modules/wct-mocha/browser.js"></script>
+  </head>
+  <body>
+    <script>
+      WCT.loadSuites([
+        'awesome-tests.js',
+        'awesomest-tests.html',
+      ]);
+    </script>
+  </body>
+</html>
+```
+
+_When you use `wct` on the command line, it is generating an index like this for
+you based on the suites you ask it to load._
+
+
+
+# Nitty Gritty
+
+## Polymer
+
+By default, WCT will defer tests until `WebComponentsReady` has fired. This
+saves you from having to wait for elements to upgrade and all that yourself.
```

Comment:
Add a little more info about WebComponentsReady? My understanding is that the polyfills emit this event. What if you aren't using the polyfills? Does Polymer fire it (I genuinely don't know)?

Reply:
TBH this is copypasta from WCT's README with changes made only to deal with the specific wct-mocha stuff vs wct/wct-browser-legacy.

I have updated the README to be much more explicit now about how this occurs.  FWIW its the @webcomponents/webcomponentsjs/webcomponents-loader.js (and/or its bundles) that emit this.